CMakeLists.txt

```diff
@@ -140,18 +141,27 @@ if(BUILD_DEPENDENCIES)
   if (LINK_PROFILER)
     build_dependency(gperftools)
   endif()
+endif()
+
+if(BUILD_AWS_DEPENDENCIES OR BUILD_DEPENDENCIES)
```

Comment:
Perhaps I am not understanding the change but as is it makes no sense to me. In case of specifying both of the flags, the same codepath will be executed.

The SDK cmake file is capable to simply using the pre-existing libraries by not specifying BUILD_DEPENDENCIES and supplying OPEN_SRC_INSTALL_PREFIX

Reply:
I understand I could do things that way, but if I have BUILD_DEPENDENCIES off and use OPEN_SRC_INSTALL_PREFIX, that requires building open source parts ahead of time. I would like the whole setup to be automated, so that I can use FetchContent to include this in a larger project and automatically build. Adding a variable helped accomplish this in my use case, but I understand if it's niche enough that you don't want to merge

Reply:
Understood. As you mentioned, this might not be generic enough to be included in the main repository. Some of our customers simply run two-stage builds specifying different parameters while others do indeed build the dependencies or fetch them from platform specific repositories ahead of the time to bake into the image.